src/libs/printer.js

```diff
@@ -116,23 +116,27 @@ export function printPegjsAst(path, options, print) {
     switch (node.type) {
         case "grammar":
             // This is the root node of a Pegjs grammar
-            body = join(concat([hardline, hardline]), path.map(print, "rules"));
+            // A `hardline` is inserted at the end so that any trailing comments
+            // are printed
+            body = [join(concat([hardline, hardline]), path.map(print, "rules")), hardline];
+
+            if (node.initializer) {
+                body.unshift(
+                    path.call(print, "initializer"),
+                    hardline,
+                    hardline
+                );
+            }
 
-            if (!node.initializer) {
-                // A `hardline` is inserted at the end so that any trailing comments
-                // are printed
-                return concat([body, hardline]);
+            if (node.ginitializer) {
+                body.unshift(
+                    path.call(print, "ginitializer"),
+                    hardline,
+                    hardline
+                );
             }
 
-            // A `hardline` is inserted at the end so that any trailing comments
-            // are printed
-            return concat([
-                path.call(print, "initializer"),
-                hardline,
-                hardline,
-                body,
-                hardline,
-            ]);
+            return concat(body);
```

Comment:
Prettier is deprecating `concat` and just using arrays. Can you remove the `concat`?